assets/app/scripts/directives/logViewer.js

```diff
@@ -11,13 +11,12 @@ angular.module('openshiftConsole')
     'logLinks',
     function($sce, $timeout, $window, AuthService, APIDiscovery, DataService, logLinks) {
 
-      // Create a template for each log line that we clone below.
-      var logLineTemplate = $('<div row class="log-line"/>');
-      $('<div class="log-line-number"><div row flex main-axis="end"></div></div>').appendTo(logLineTemplate);
-      $('<div flex class="log-line-text"/>').appendTo(logLineTemplate);
-
       // Keep a reference the DOM node rather than the jQuery object for cloneNode.
```

Comment:
add a comment here that changes to this template require modifying `addLine()` below